source/reference/minio-mc-admin/mc-admin-prometheus-metrics.rst

```diff
@@ -63,146 +74,125 @@ Parameters
+      Valid values for metrics version 3 are:
 
-         ``resource`` metrics added
+      - ``api``
+      - ``audit``
+      - ``cluster``
+      - ``debug``
+      - ``ilm``
+      - ``logger``
+      - ``notification``
+      - ``replication``
+      - ``scanner``
+      - ``system``
+
+      If not specified, a ``v3`` command returns all metrics.
 
-      Valid values are:
+      Valid values for metrics version 2 are:
 
       - ``bucket``
       - ``cluster``
       - ``node``
       - ``resource``
 
-      If not specified, the command returns cluster metrics.
-      Cluster metrics include rollups of certain node metrics.
-      The output includes additional information about each metric, such as if its value is a ``counter`` or ``gauge``.
+      If not specified, a ``v2`` command returns cluster metrics.
+      Cluster metrics also include rollups of certain node metrics.
+
 
-Global Flags
+Global flags
 ~~~~~~~~~~~~
 
 .. include:: /includes/common-minio-mc.rst
    :start-after: start-minio-mc-globals
    :end-before: end-minio-mc-globals
 
 
-Example
--------
+Examples
+--------
+
+Print v3 metrics
+~~~~~~~~~~~~~~~~
+
+Use :mc-cmd:`mc admin prometheus metrics --api-version v3 <mc admin prometheus metrics --api-version>` to print all available v3 metrics and their current values for a MinIO deployment:
 
-Print bucket metrics
-~~~~~~~~~~~~~~~~~~~~
+.. code-block:: shell
+   :class: copyable
+
+   mc admin prometheus metrics ALIAS --api-version v3
+
+- Replace ``ALIAS`` with the :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+To print a specific type of metrics, include the :mc-cmd:`~mc admin prometheus metrics TYPE`.
+The following prints all scanner metrics for a deployment:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin prometheus metrics ALIAS scanner --api-version v3
+
+Print v3 bucket replication metrics
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Use :mc-cmd:`mc admin prometheus metrics` to print bucket metrics for a MinIO deployment:
+Certain metric types accept a :mc-cmd:`~mc admin prometheus metrics --bucket` parameter to specify the bucket for which to print metrics.
+The following example prints v3 replication metrics for bucket ``mybucket``:
 
 .. code-block:: shell
    :class: copyable
 
-      mc admin prometheus metrics ALIAS bucket
+   mc admin prometheus metrics ALIAS replication --bucket mybucket --api-version v3
 
 - Replace ``ALIAS`` with the :mc-cmd:`alias <mc alias>` of the MinIO deployment.
 
-The output resembles the following:
+To print API metrics for a bucket, replace ``replication`` with ``api``.
+
+Print v2 metrics
+~~~~~~~~~~~~~~~~
+
+By default, :mc-cmd:`mc admin prometheus metrics` prints v2 cluster metrics:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin prometheus metrics ALIAS
+
+- Replace ``ALIAS`` with the :mc-cmd:`alias <mc alias>` of the MinIO deployment.
+
+To print another type of metrics, specify the desired :mc-cmd:`~mc admin prometheus metrics TYPE`.
```

Comment:
I'd split this out to a separate example and maybe change line 180 to be `Print v2 cluster metrics`.